### DISCUSSION TITLE
Overlap in banners / messages?

### DISCUSSION THREAD

**Discussion by daniel-ac-martin** [UKHomeOffice/design-system#342] [Ideas]
It might be worth thinking about whether any of the following components could be merged? (i.e. Do they have overlapping purposes?)
https://design.homeoffice.gov.uk/components?name=Alert
https://design.homeoffice.gov.uk/components?name=Expandable%20banner
https://design.homeoffice.gov.uk/components?name=Status%20message

It might be that we should merge the functionality of one or more.

Then again, perhaps they have different purposes? (I defer to the interaction designers!)

**Comment by daniel-ac-martin** [comment]
@Sarraounia: Is this relevant to what you've been looking at recently?

**Comment by isobelseacombe** [comment]
I think that we need to more clearly define the uses of these components (which I think Ounie has been doing). My assumption is that they have different uses and can't all be merged into one. 

I started documenting some uses on this discussion about highlighting important information: https://github.com/UKHomeOffice/design-system/discussions/336

**Reply by isobelseacombe** [reply]
Perhaps the expandable banner and status could be merged?

**Comment by Sarraounia** [comment]
@daniel-ac-martin not sure what the advantage is of merging them. Looks to me like they are used in different services and for different scenarios. Is there evidence that they way we've got it at the moment is confusing for people? 

@isobelseacombe I've drafted some text in #358 for how to write content for an alert. Not defining the use exactly, just how to format content in the banner.

**Reply by daniel-ac-martin** [reply]
They just seemed to be solving the same problem to me. (But I'm not a designer!)

If they are solving different problems, and it's clear to others when to use each, then there's nothing to be done here.

**Comment by tomyems** [comment]
Merged into https://github.com/UKHomeOffice/design-system/discussions/336.
